modules/rostests/apitests/comctl32/propsheet.c

```diff
@@ -73,49 +74,98 @@ Page1DlgProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
     return 0;
 }
 
+#ifdef _MSC_VER
+#define CHECK_STRUCT_SIZE(x, y) C_ASSERT((x) == (y))
+#else
+// Can't do this compile time, thanks gcc
+// 'error: non-nested function with variably modified type'
```

Comment:
Hm yeah maybe this could be done outside of the "test_StructSizes()" (since then this would be checked at compile time, and not during runtime).

Reply:
nope, same problem (tried that first).